torchrl/envs/vec_env.py

```diff
@@ -1076,3 +1089,257 @@ def _run_worker_pipe_shared_mem(
             else:
                 # don't send env through pipe
                 child_pipe.send(("_".join([cmd, "done"]), None))
+
+
+class MultiThreadedEnvWrapper(_EnvWrapper):
+    """Wrapper for envpool-based multithreaded environments."""
+
+    _verbose: bool = False
+
+    def __init__(
+        self,
+        env: Optional["envpool.python.envpool.EnvPoolMixin"] = None,
+        **kwargs,
+    ):
+        if not _has_envpool:
+            raise ImportError(
+                f"""envpool python package was not found. Please install this dependency.
```

Comment:
nitpicking but this error could happen both because `envpool` is missing or because `treevalue` is missing. Maybe we should mention both in the error

Reply:
Good point, adjusted the error message